offline/upgrading.md

```diff
@@ -135,16 +169,49 @@ d kubectl get nodes -owide
 ```
 They should all report ready.
 
-## Deploying wire-server using helm
+If you are worried about disk space, removing unneeded images could be performed here.
+
+## Upgrading wire-server using helm
 
-It's now time to upgrade the helm charts on top of kubernetes, upgrading the Wire platform.
+### Upgrading non-wire components:
 
-inspect your values and secrets files, comparing them to the new defaults.
+Copy your external service definition values into place.
+```
+```
 
-Now deploy `wire-server`:
+First, upgrade the external service definitions, as those rarely change.
+```
+d helm upgrade cassandra-external ./charts/cassandra-external/ --values ./values/cassandra-external/values.yaml
+d helm upgrade elasticsearch-external ./charts/elasticsearch-external/ --values ./values/elasticsearch-external/values.yaml
+d helm upgrade minio-external ./charts/minio-external/ --values ./values/minio-external/values.yaml
+```
 
+Next, upgrade the internal non-wire services.
 ```
-d helm install wire-server ./charts/wire-server --timeout=15m0s --values ./values/wire-server/values.yaml --values ./values/wire-server/secrets.yaml
+d helm upgrade fake-aws ./charts/fake-aws/ --values ./values/fake-aws/prod-values.example.yaml
+d helm upgrade databases-ephemeral ./charts/databases-ephemeral/ --values ./values/databases-ephemeral/values.yaml
+d helm upgrade reaper ./charts/reaper/
+```
+
+Finally, upgrade demo-smtp. note that you may have to look for a values.yaml, instead of a prod-values.example.yaml file, in the case that demo-smtp was configured. if it's not present, use prod-values.example.yaml.
```

Comment:
```suggestion
Finally, upgrade demo-smtp. Note that you may have to look for a `values.yaml`, instead of a `prod-values.example.yaml` file, in the case that demo-smtp was configured. If it's not present, use `prod-values.example.yaml`.
```